src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,114 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+/**
+ * User model is using to save users data
+ *
+ * @author Jorge Díaz
+ * @version 1.0.0
+ */
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString
+public class User {
+
+    /**
+     * User's auto generated identification
+     */
+    @Id
+    @GeneratedValue
+    @Setter(AccessLevel.NONE)
+    private long id;
+
+    /**
+     * Identification's name
+     */
+    @Column(nullable = false, unique = true)
+    private String username;
+
+    /**
+     * User's name
+     */
+    @Column(nullable = false)
+    private String name;
+
+    /**
+     * User's birth date
+     */
+    @Column(nullable = false)
+    private LocalDate birthDate;
+
+    /**
+     * Books that belongs to the user
+     */
+    @ManyToMany(cascade = {CascadeType.MERGE, CascadeType.REFRESH})
+    @JoinTable(name = "users_books",
```

Comment:
not use joinTable, please cheack this link.
https://stackoverflow.com/questions/5478328/jpa-jointable-annotation